Doc/README.rst

```diff
@@ -8,6 +8,11 @@ On Ubuntu, you can install these with the following commands::
 
     sudo apt install hevea texlive-latex-base texlive-latex-extra python-pygments
 
+On Fedora (tested for Workstation 33):
```

Comment:
You need a double colon to indicate an RST literal block

Reply:
Mb. Fixed it in new commit.